### DISCUSSION TITLE
[Question] How do I disable wallbash specifically for kitty?

### DISCUSSION THREAD

**Discussion by Radrahil** [prasanthrangan/hyprdots#1000] [Q&A]
See title

**Comment by kRHYME7** [comment]
rename ```  ~/.config/hypr/wallbash/kitty.dcol ``` or just remove the ".dcol " suffix 


you can also run ``` Hyprdots wallbash term -t ``` to Toggle the extra terminal color presets.

**Comment by Radrahil** [comment]
It doesn't stop wallbash from affecting kitty? I specifically want it to not apply for kitty

**Comment by kRHYME7** [comment]
> It doesn't stop wallbash from affecting kitty? I specifically want it to not apply for kitty

Do this then:

##### Warning: Do this while wallbash is disabled. 
 rename ```  ~/.config/hypr/wallbash/kitty.dcol  ```  to remove the ".dcol " suffix. The .dcol keyword makes this file execute, something which triggers wallbash on kitty. By removing this you can then disable kitty's wallbash. 



I'll try to add an option like this soon. 

```  Hyprdots wallbash  [package e.g kitty]  0  ``` invoking this will automate the commands above.

**Comment by kRHYME7** [comment]
@Radrahil  just added a new option for in hyprdots-ctl repo. 

``` Hyprdots wallbash kitty 0 ``` To disable , 1 to enable. ``` Hyprdots wallbash kitty ``` for help. 
Before doing this please make sure your reverted the steps I give above. This will ensure that we have the files properly set.


>rename  ~/.config/hypr/wallbash/kitty.dcol to remove the ".dcol " suffix. The .dcol keyword makes this file execute, something which triggers wallbash on kitty. By removing this you can then disable kitty's wallbash.

**Comment by Radrahil** [comment]
I've noticed that switching wallpapers (not themes) seems to ignore this effect and then I have to turn it off and on again. However, this only happens when switching the wallpaper while wallbash is enabled. Re enabling wallbash after switching keeps theme as expected

**Comment by kRHYME7** [comment]
~Sorry I didn't catch it, can you explain more?.~

I saw it now. It should work now.  

before updating 

run first  ``` Hyprdots wallbash kitty 1 ``` 

then ``` yay -Sy hyprdots-ctl ```

It should work now. Thanks for the heads up

**Comment by keremimo** [comment]
I'm trying to get wallbash to stop modifying my VS Code and there is no Hyprdots. I checked Hyde but could not figure out how to disable wallbash. Did this change? I don't even mind if I have to completely disable wallbash if it's an option.

**Reply by mrmostafaxd** [reply]
go to `$HOME/.local/share/bin/wallbashcode.sh`, and remove/comment the following lines:
```sh
if [ "${extTheme}" != "wallbash" ] ; then
    jq '.["workbench.colorTheme"] = "wallbash"' "${codeConf[i]}/User/settings.json" > "${tmpFile}" && mv "${tmpFile}" "${codeConf[i]}/User/settings.json"
fi
```

**Reply by kRHYME7** [reply]
@Keremimo  

remove code.dcol or rename it into any suffix except '.dcol'  
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/022154b2-4377-4bf0-abe3-9d63ff0c6a14)



for disabling wallbash run ` wallbashtoggle.sh -m ` then select theme
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/48d53c92-c3a9-45b7-b13b-954bccd83cad)



in CLI, you can use this  ` Hyde wallbash`  command
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/2ec52cb4-c7e2-46eb-80d6-bcf58d78d941)

**Reply by keremimo** [reply]
That did the trick, thanks!
